npm-scripts.js

```diff
@@ -120,13 +120,34 @@ switch (task)
 	{
 		if (!process.env.MEDIASOUP_WORKER_BIN)
 		{
+			if (isWindows)
```

Comment:
This is needed not only in `postinstall`, but every time worker is built, hence should likely be in `worker:build` instead, as well as in `test:worker`. Cleanup should stay here though.

Reply:
I've added a separate task `install-msys-make` for make installation (since I doubt that those who need to run `test:worker` or `worker:build` outside of `postinstall` would like do download everything each time they run the tasks.)